converter/biostudies.py

```diff
@@ -104,10 +108,10 @@ def _parse_name(*args):
                 "name": ['', default_to, "Phone"],
                 "value": ['phone']
             },
-            {
-                "name": ['', default_to, "affiliation"],
-                "value": ['institution']
-            },
+            # {
```

Comment:
Why the comment?
If it is not needed, it can be deleted.

Reply:
You are right, I forgot to remove it. Tx for notifying me

Reply:
Removed those lines